docs/en/siem/detection-engine-intro.asciidoc

```diff
@@ -77,25 +77,50 @@ To investigate a signal in the Timeline, click the *View in timeline* icon.
 [[detections-permissions]]
 === {kib} and index privileges required for Detections
 
-[IMPORTANT]
+Depending on your privileges and whether a `.siem-signals-<space name>` index 
+has been created for the {kib} space, you might see an error message when you 
+try to open the *Detections* page.
+
+[NOTE]
```

Comment:
Thanks for adding this note @benskelker! 🙏 
Please consider moving this note to the `Detection engine permissions required` section below, because running a local deployment without security enabled will result in the `Detection engine permissions required` message being displayed.